material/problems/VectorGeometry/FirstAndTen/FirstAndTen.tex

```diff
@@ -1,116 +1,240 @@
+  \plus (20\mathrm{~yards})~\uvecz{n}$.
+  %
+  \\[0.45pc] Finally, let the football be identified as point $Q$.
 \end{minipage}
 \hfill
 \begin{minipage}[t]{0.27\linewidth}
-\vspace{-0.5pc}
-\includegraphics[width=\linewidth]{firstAndTen.jpg}
+  \vspace{-0.5pc}
+  \includegraphics[width=\linewidth]{firstAndTen.jpg}
 \end{minipage}
 \\[0.2pc]
 %
 \begin{enumerate}
-\item
-%\\[0.5pc]
-\begin{minipage}[t]{0.6\linewidth}
-When the football is lying on the field, the operator clicks on the football in the video.
-Assume the click is converted into a direction vector pointing to $Q$ from $B_o$ as $\uvecWithHat{u}^{Q/B_o} = 1~\uvecx{b}$.
-\\[0.5pc]
-Find a \textbf{system of \boldUnderlineDarkRed{scalar} equations} that can be used to solve for the \underline{\uvecx{n} and \uvecy{n} measures} of the football's position from $N_o$. The rotation table will be formed later; for now you may leave unknown dot-products un-evaluated (such as $\uvecx{b} \cdot \uvecx{n}$).
-\\[0.5pc]
-To do this, it may help to:
-\begin{itemize}
-\item Introduce \boldDarkRed{unknown} measures for $Q$'s position from $N_o$ to help you express \posvec{N_o}{Q}.
-\item Write \posvec{B_o}{Q} as an \boldDarkRed{unknown} multiple of $\uvecWithHat{u}^{Q/B_o}$.
-\item Write a \boldDarkRed{vector loop} equation.
-\end{itemize}
-\end{minipage}
-\hfill
-\begin{minipage}[t]{0.35\linewidth}
-\flushright
-\vspace*{0pt}
-\includegraphics[width=\linewidth]{football.png}
+  \item
+  %\\[0.5pc]
+  \begin{minipage}[t]{0.6\linewidth}
+    When the football is lying on the field, the operator clicks on the
+    football in the video. Assume the click is converted into a direction
+    vector pointing to $Q$ from $B_o$ as
+    $\uvecWithHat{u}^{Q \division B_o} \equals 1~\uvecx{b}$.
+    \\[0.5pc]
+    Find a \textbf{system of \boldUnderlineDarkRed{scalar} equations} that can be
+    used to solve for the \underline{\uvecx{n} and \uvecy{n} measures} of the
+    football's position from $N_o$. The rotation table will be formed later; for
+    now you may leave unknown dot-products un-evaluated
+    (such as $\uvecx{b} \cdot \uvecx{n}$).
+    \\[0.5pc]
+    To do this, it may help to:
+      \begin{itemize}
+        \item Introduce \boldDarkRed{unknown} measures for $Q$'s position
+        from $N_o$ to help you express \posvec{N_o}{Q}.
+        \item Write \posvec{B_o}{Q} as an \boldDarkRed{unknown} multiple of
+        $\uvecWithHat{u}^{Q \division B_o}$.
+        \item Write a \boldDarkRed{vector loop} equation.
+      \end{itemize}
+  \end{minipage}
+  \hfill
+\begin{minipage}[t]{0.44\linewidth}
+  \minipageTopAnchor
+  %\vspace*{-1cm}
+  \begin{center}
+    \includegraphicsAB[width=0.66\linewidth]
+      {football1.png}
+      {football.png}
+  \end{center}
 \end{minipage}
-
+\Solution{}{0.99\linewidth}{
+  \begin{minipage}{0.65\linewidth}
+     \begin{tabular}{@{ }l@{}r@{ \equals[\;] }ll}
+       &\posvec{N_o}{Q}& $x\uvecx{n} \plus y\uvecy{n} \plus o\uvecz{n}$
```

Comment:
space after &.
Also, it looks like you put an `o` in a bunch of places where it was supposed to be `0`. :)